pepper/cli.py

```diff
@@ -322,6 +326,21 @@ def parse_cmd(self):
                 logger.error("Invalid JSON given.")
                 raise SystemExit(1)
 
+        if self.options.json_file:
+            try:
+                with open(self.options.json_file, 'r') as json_content:
```

Comment:
It would be great to use stdin instead of file if parameter is `-`

Reply:
Yes but this feature can be good using --json parameter not --json-file